data/data/aws/vpc/sg-worker.tf

```diff
@@ -56,6 +56,36 @@ resource "aws_security_group_rule" "worker_ingress_vxlan_from_master" {
   to_port   = 4789
 }
 
+resource "aws_security_group_rule" "worker_ingress_geneve" {
+  type              = "ingress"
+  security_group_id = "${aws_security_group.worker.id}"
+
+  protocol  = "udp"
+  from_port = 6081
+  to_port   = 6081
+  self      = true
+}
+
+resource "aws_security_group_rule" "worker_ingress_geneve_from_master" {
+  type                     = "ingress"
+  security_group_id        = "${aws_security_group.worker.id}"
+  source_security_group_id = "${aws_security_group.master.id}"
+
+  protocol  = "udp"
+  from_port = 6081
+  to_port   = 6081
+}
+
+resource "aws_security_group_rule" "worker_ingress_ovnsb_from_master" {
```

Comment:
pretty sure you don't need this rule; the ovn databases should be running on master nodes

Reply:
Fixed.